pacemaker.spec.in

```diff
@@ -176,6 +176,9 @@ BuildRequires: python-devel >= 2.6
 Requires:      libqb >= 0.13.0
 BuildRequires: libqb-devel >= 0.13.0
 
+# Basics required for the build (even if usually satisfied through other BRs)
+BuildRequires: coreutils findutils grep sed
```

Comment:
BTW when adding new dependencies, we should check whether the package names are the same on suse. The GNUmakefile target for *-suse.spec actually uses sed to replace package names (a questionable approach, but one we want to stick with for the short term).